scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/forms/DatasetEditForm.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.diging.scriptoocloud.web.forms;
+
+import edu.asu.diging.scriptoocloud.web.validation.FieldNotSame;
+
+import javax.validation.constraints.NotEmpty;
+
+@FieldNotSame(first = "oldName", second = "newName", message = "New Dataset name is the same as the old Dataset name")
+public class DatasetEditForm {
+
+    private Long id;
+    private int index;
+    private String oldName;
+    @NotEmpty(message = "Dataset name cannot be empty")
+    private String newName;
```

Comment:
Just show a form with all the fields that can be changed (right now just the name I guess). If we extend the properties of a dataset, we would otherwise have to add to form fields for each new field and it doesn't provide any additional benefit really.